### PR TITLE
Exit editmode with both `esc` and `tab`

### LINKED ISSUE #1232: Allow <Tab> to exit editmode
## Feature Description

Pressing <kbd>tab</kbd> should (also) exit edit mode, just as <kbd>esc</kbd> does today.

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by barklund**
Fixes #1232.

### CODE REVIEW

File: assets/src/edit-story/components/canvas/editLayer.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ function EditLayerForElement({ element }) {
 
   const focusCanvas = useFocusCanvas();
 
-  useKeyDownEffect(ref, { key: 'esc', editable: true }, () => clearEditing(), [
+  const key = ['esc', 'tab'];
```
Comment (dvoytenko) [inline assets/src/edit-story/components/canvas/editLayer.js:80]: Hmm. This definitely works for text, but does it work for image editor? E.g. we may want to tab between the pannable image and the scale slider?

**Review by dvoytenko — CHANGES_REQUESTED** [review]

File: assets/src/edit-story/components/canvas/editLayer.js (hunk 1 above)
Comment (barklund) [reply assets/src/edit-story/components/canvas/editLayer.js:80]: The issue comes straight from Fishfood. @pbakaus, what's your take on this? Would users want to tab in editmode for images and expect it to do other things than exit?

Comment (pbakaus) [reply assets/src/edit-story/components/canvas/editLayer.js:80]: I think the main question is whether there's another way to get to the scale slider using the keyboard...if not, then I think it makes sense to revert course and not modify this behavior (so we treat edit mode as a state that captures focus, and you always need esc to get out of it).

Comment (dvoytenko) [reply assets/src/edit-story/components/canvas/editLayer.js:80]: My first question is why doesn't "focusout" doesn't work?

**Comment by swissspidy** [comment]
Closing since #1232 was closed. Reopen & ping on #1232 if that's incorrect.

**Pull request closed without merging**